### ISSUE TITLE
native_pdf_view depends on device_info error

### ISSUE THREAD

**Issue by zs-dima** [rbcprolabs/packages.flutter#134]
>   device_info: ^1.0.0
- lead to:

```
Because native_pdf_view 3.8.0 depends on device_info ^0.4.2+4 and no versions of native_pdf_view match >3.8.0 <4.0.0, native_pdf_view ^3.8.0 requires device_info ^0.4.2+4.
So, because jkapp depends on both device_info ^1.0.0 and native_pdf_view ^3.8.0, version solving failed.
pub get failed (1; So, because jkapp depends on both device_info ^1.0.0 and native_pdf_view ^3.8.0, version solving failed.)
```

**Comment by xinlangzi** [comment]
@zs-dima 

Has anyone overcame this problem?

**Comment by SergeShkurko** [comment]
fixed in 3.9.0
